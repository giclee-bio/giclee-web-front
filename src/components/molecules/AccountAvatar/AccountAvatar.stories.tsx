import { Meta, StoryObj } from '@storybook/react'

import AccountAvatar from './AccountAvatar'

const meta: Meta<typeof AccountAvatar> = {}

export default meta
type Story = StoryObj<typeof AccountAvatar>

export const Index: Story = {
  args: {},
}
