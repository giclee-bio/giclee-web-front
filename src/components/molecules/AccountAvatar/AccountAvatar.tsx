import Avatar from '@/components/atoms/Avatar'

const AccountAvatar = () => {
  return (
    <>
      <div>
        <button>
          <Avatar alt='aa' size='medium' src='/nacal.png' />
          <p className='mt-2 text-sm'>変更する</p>
        </button>
      </div>
    </>
  )
}

export default AccountAvatar
