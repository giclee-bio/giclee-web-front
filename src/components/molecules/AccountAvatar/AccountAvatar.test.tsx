import { render } from '@testing-library/react'

import AccountAvatar from './AccountAvatar'

describe('AccountAvatar', () => {
  it('snapshot', async () => {
    const { asFragment } = render(<AccountAvatar />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
