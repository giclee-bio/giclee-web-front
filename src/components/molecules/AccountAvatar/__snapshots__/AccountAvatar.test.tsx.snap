// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountAvatar snapshot 1`] = `
<DocumentFragment>
  <div>
    <button>
      <img
        alt="aa"
        class="rounded-full"
        data-nimg="1"
        decoding="async"
        height="80"
        loading="lazy"
        src="/_next/image?url=%2Fnacal.png&w=256&q=75"
        srcset="/_next/image?url=%2Fnacal.png&w=96&q=75 1x, /_next/image?url=%2Fnacal.png&w=256&q=75 2x"
        style="color: transparent;"
        width="80"
      />
      <p
        class="mt-2 text-sm"
      >
        変更する
      </p>
    </button>
  </div>
</DocumentFragment>
`;
